### PR TITLE
New package: rpm-4.12.0

### PR DESCRIPTION
**Pull request by renatoaguiar**

### CODE REVIEW

File: srcpkgs/rpm/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+makedepends="zlib-devel nss-devel file-devel popt-devel libarchive-devel db-devel lua52-devel"
+short_desc="The RPM Package Manager"
+maintainer="Renato Aguiar <contact@renatoaguiar.org>"
+license="GPL-3"
+homepage="http://rpm.org"
+distfiles="http://rpm.org/releases/rpm-4.12.x/${pkgname}-${version}.tar.bz2"
+checksum=770d5cd1aa9430affa6ddd336a81dae79704962478f6b336d5b694c3e02113a2
+
+do_configure() {
+	./configure ${configure_args} CPPFLAGS="$(pkg-config --cflags nss)"
```
Comment (Vaelatern) [inline srcpkgs/rpm/template]: Suggest if you need to add `$(pkg-config --cflags nss)` to the `$CPPFLAGS` you do so outside a function, replacing the function with the declaration `CPPFLAGS=$(pkg-config --cflags nss)`

Comment (Duncaen) [inline srcpkgs/rpm/template]: Are you sure the dependencies are already installed at this time ;)?

Comment (Vaelatern) [inline srcpkgs/rpm/template]: Ah, I suppose not. Perhaps in a `pre_configure` then?

**Comment by chneukirchen** [comment]
4.13.0.1 is current, please update and reopen.

**Pull request closed without merging**